CHEF-32264 - Chef Workstation 25.14 release notes (#4639)
chef/chef-web-docs: content/release_notes/workstation.md

<!-- cSpell:disable  -->
<!-- vale off -->

## Chef Workstation 25.14

Release Date: May 21, 2026.

### Component updates

- Updated cookstyle to v8.6.10
- Updated berkshelf to v8.1.21
- Updated chef-vault to v4.2.9
- Updated fauxhai-chef to v9.4.20

> - Updated test-kitchen to v4.0.0, which adds support for authenticated downloads from the new download API.

Reviewer: ```suggestion
- Updated test-kitchen to v4.0.0, which moves the provisioner to it's own gem.
- Added kitchen-omnibus-chef gem for the chef provisioner and adds support for authenticated downloads from the new download API.
```

- Using custom gems or changing the Ruby path isn't supported in this release.
- The `knife google server create` command on Windows can fail during password reset if the GCEAgent initializes slowly (for example, on slow boot or low-resource machines).

> ## Chef Workstation 25.14

Reviewer 2: Reminder to add full version number.

Author: Updated to 25.14.1

Release date: May 21, 2026

### Component updates

- Updated Cookstyle to v8.6.10.
- Updated Berkshelf to v8.1.21.
- Updated Chef Vault to v4.2.9.
- Updated Fauxhai to v9.4.20.
- Updated Test Kitchen to v4.0.0, which moves the provisioner to its own gem.
- Added the kitchen-omnibus-chef gem, which provides the Chef provisioner for Test Kitchen and supports authenticated downloads from the Chef downloads API.
- Replaced winrm, winrm-elevated, and winrm-fs gems with chef-winrm, chef-winrm-elevated, and chef-winrm-fs.

Updated the following Test Kitchen drivers:

 - Updated kitchen-ec2 to v3.22.3.
 - Updated kitchen-digitalocean to v0.16.2.
 - Updated kitchen-google to v2.6.2.
 - Updated kitchen-openstack to v7.0.1.
 - Updated kitchen-vagrant to v2.2.1.
 - Updated kitchen-vcenter to v2.12.3.

### Security

- Updated curl from v8.16.0 to v8.19.0.
- Updated Go from v1.23.12 to v1.26.2.
- Updated git-windows from v2.48.1 to v2.54.0.
- Updated activesupport from v7.2.3 to v7.2.3.1.

> - Updated json from v2.18.1 to v2.19.4.

Reviewer 2: Is json updated to v2.19.5?

Author: Yes now it got updated to 2.19.5, I have updated here as well. Thanks

- Updated rack from v3.2.5 to v3.2.6.
- Updated chef-analyze from v0.1.192 to v0.1.194.
- Updated golang.org/x/text from v0.3.8 to v0.24.0.
- Updated golang.org/x/crypto from v0.31.0 to v0.45.0.

> 

Reviewer 2: Should this include go-retryablehttp v0.7.2 updated to v0.7.7?

Author: No @IanMadd , I dont think we need to include this since its not a direct dependency. Its coming from some other lib and we are just replacing the version here.

### Bug fixes

- Fixed REXML XPath deprecation warnings that appeared during Test Kitchen converge operations when using the Azure driver on Windows platforms.

> - Fixed "Exception in finalizer" and "can't alloc thread (ThreadError)" warnings that appeared after Test Kitchen completed when using the vCenter driver with Windows targets.

Reviewer: This was from the winrm gem and affected more than just the vcenter driver.
```suggestion
- Fixed "Exception in finalizer" and "can't alloc thread (ThreadError)" warnings that appeared after Test Kitchen run against Windows targets.
```
